### ISSUE TITLE
CMake warning

### ISSUE THREAD

**Issue by zapashcanon** [LMMS/lmms#3985]
Hi,

When using:

```sh
cmake .. -DCMAKE_INSTALL_PREFIX=../target/ -DWANT_QT5=ON
```

I got:

```txt
[...]
CMake Deprecation Warning at CMakeLists.txt:11 (CMAKE_POLICY):
  The OLD behavior for policy CMP0026 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
[...]
```

**Comment by lukas-w** [comment]
For the record, this is what [`CMP0026`](https://cmake.org/cmake/help/v3.0/policy/CMP0026.html) is about:

> Disallow use of the LOCATION property for build targets.
> 
> CMake 2.8.12 and lower allowed reading the LOCATION target property (and configuration-specific variants) to determine the eventual location of build targets. This relies on the assumption that all necessary information is available at configure-time to determine the final location and filename of the target. However, this property is not fully determined until later at generate-time. At generate time, the $<TARGET_FILE> generator expression can be used to determine the eventual LOCATION of a target output.
> 
> Code which reads the LOCATION target property can be ported to use the $<TARGET_FILE> generator expression together with the file(GENERATE) subcommand to generate a file containing the target location.
> 
> The OLD behavior for this policy is to allow reading the LOCATION properties from build-targets. The NEW behavior for this policy is to not to allow reading the LOCATION properties from build-targets.


I don't think we're using the `LOCATION` property anywhere, so this should be safe to set to `NEW`. The only use of `LOCATION` was removed via 65919749fed576ed2b3a8d2785bd0290d238b6e9 (#1891).

**Comment by tresf** [comment]
Closed via 79e4775.
